components/TextFields/src/MDCTextInputControllerFilled.m

```diff
@@ -265,6 +265,10 @@ - (void)updatePlaceholder {
   }
 }
 
+- (UIEdgeInsets)textInsets:(UIEdgeInsets)defaultInsets {
```

Comment:
Nit: As we discussed, we believe subclass implementations don't need to be overridden. Please remove if possible.